docs/tutorial/anonlink-novt.ipynb

```diff
@@ -0,0 +1,942 @@
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>recid</th>\n",
+       "      <th>givenname</th>\n",
+       "      <th>surname</th>\n",
+       "      <th>suburb</th>\n",
+       "      <th>postcode</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>1503359</td>\n",
+       "      <td>pauline</td>\n",
+       "      <td>camkbell</td>\n",
+       "      <td>lilescille</td>\n",
+       "      <td>28091</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1972058</td>\n",
+       "      <td>deborah</td>\n",
+       "      <td>galyen</td>\n",
+       "      <td>ennike</td>\n",
+       "      <td>286z3</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>889525</td>\n",
+       "      <td>charle5</td>\n",
+       "      <td>mitrhell</td>\n",
+       "      <td>roaring river</td>\n",
+       "      <td>28669</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>4371845</td>\n",
+       "      <td>petehr</td>\n",
+       "      <td>werts</td>\n",
+       "      <td>swannanoa</td>\n",
+       "      <td>28478</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>1187991</td>\n",
+       "      <td>katpy</td>\n",
+       "      <td>silbiger</td>\n",
+       "      <td>duyham</td>\n",
+       "      <td>27705</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     recid givenname   surname         suburb postcode\n",
+       "0  1503359   pauline  camkbell     lilescille    28091\n",
+       "1  1972058   deborah    galyen         ennike    286z3\n",
+       "2   889525   charle5  mitrhell  roaring river    28669\n",
+       "3  4371845    petehr     werts      swannanoa    28478\n",
+       "4  1187991     katpy  silbiger         duyham    27705"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "corruption_rate = 20\n",
+    "file_template = 'data/ncvr_numrec_5000_modrec_2_ocp_' + str(corruption_rate) + '_myp_{}_nump_10.csv'\n",
+    "df1 = pd.read_csv(file_template.format(0))\n",
+    "df1.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A hashing schema instructs clkhash how to treat each column for generating CLKs. A detailed description of the hashing schema can be found in the api docs. We will ignore the columns ‘ENTID’ for CLK generation."
```

Comment:
add a link to where to find the hashing schema docs.

The first column is called `recid` in the dataset.